aeron-client/src/main/java/io/aeron/Aeron.java

```diff
@@ -1313,8 +1314,8 @@ public void close()
         {
             final MappedByteBuffer cncByteBuffer = this.cncByteBuffer;
             this.cncByteBuffer = null;
-            IoUtil.unmap(cncByteBuffer);
-            super.close();
+            AeronCloseHelper.close(errorHandler, () -> IoUtil.unmap(cncByteBuffer));
```

Comment:
Unmap should not generate an error.

Reply:
I change all of the unmap calls to be naked.